gym/wrappers/monitoring/video_recorder.py

```diff
@@ -93,6 +103,27 @@ def __init__(self, env, path=None, metadata=None, enabled=True, base_path=None):
 
         self.frames_per_sec = env.metadata.get("render_fps", 30)
         self.output_frames_per_sec = env.metadata.get("render_fps", self.frames_per_sec)
+
+        # backward-compatibility mode:
+        self.backward_compatible_frames_per_sec = env.metadata.get(
+            "video.frames_per_second", 30
+        )
+        self.backward_compatible_output_frames_per_sec = env.metadata.get(
+            "video.output_frames_per_second", self.frames_per_sec
+        )
+        if self.frames_per_sec != self.backward_compatible_frames_per_sec:
+            logger.deprecation(
+                '`env.metadata["video.frames_per_second"] is marked as deprecated and will be replaced with `env.metadata["render_fps"]` '
+                "see https://github.com/openai/gym/pull/2654 for more details"
+            )
+            self.frames_per_sec = self.backward_compatible_frames_per_sec
```

Comment:
These assignment will create a new bug if an user implements only new `render_fps`.
What if we use `get('render_fps')` so that when it is not available, it is `None` and then if the user doesn't have that value, then we go back to use `video.frames_per_second` (which default to 30) and warn the user.

Reply:
This was fixed in https://github.com/openai/gym/pull/2703